server/queries/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
This whole file seems really overly complicated? When we did it for the todo app I followed something only that is way simpler and more clear what's going on, if you want to see it